portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewActions.testcase

```diff
@@ -418,4 +534,184 @@ definition {
 		}
 	}
 
+	@description = "LPS-192149. Verify if the Headless Action Key is not mandatory and contains an tooltip Icon with an message"
+	@priority = 3
+	test HeadlessActionKeyIsNotMandatory {
+		task ("And clicks on Create Item Action button") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("Then the Headless Action Key field is not mandatory") {
+			AssertElementNotPresent(
+				key_fieldName = "Headless Action Key",
+				locator1 = "DataSet#MANDATORY_FIELD");
+		}
+
+		task ("And next to the Headless Action Key field title there is a tooltip icon.") {
+			AssertElementPresent(
+				key_indexNumber = 1,
+				key_text = "question-circle-full",
+				locator1 = "Icon#ANY_N");
+		}
+
+		task ("When the user hovers the tooltip icon") {
+			MouseOver(
+				key_indexNumber = 1,
+				key_text = "question-circle-full",
+				locator1 = "Icon#ANY_N");
+		}
+
+		task ("Then the message “This key allows the action to be performed when the permission is associated with it” is displayed.") {
+			AssertElementPresent(
+				key_title = "",
+				locator1 = "DataSet#FILTER_TOOLTIP",
+				value1 = "This key is used to display the action if the user has the permission for it by checking if the key is present in the data item's actions array.");
+		}
+	}
```

Comment:
Just checking the mandatory field is not enough, we should try to save and check that the action can be saved without filling the field